jobserver/urls.py

```diff
@@ -34,21 +33,11 @@
 )
 
 
-router = routers.DefaultRouter()
-router.register(r"jobs", JobViewSet, "jobs")
-router.register(r"workspaces", WorkspaceViewSet, "workspaces")
-
-
-api_patterns = [
-    path("v1/", include(router.urls)),
-    path("v2/job-requests/", JobRequestAPIList.as_view()),
-    path("v2/jobs/", JobAPIUpdate.as_view()),
-]
-
 urlpatterns = [
     path("", Index.as_view()),
     path("", include("social_django.urls", namespace="social")),
-    path("api/", include(api_patterns)),
+    path("api/job-requests/", JobRequestAPIList.as_view()),
+    path("api/jobs/", JobAPIUpdate.as_view()),
```

Comment:
Is it worth temporarily add these paths under the `v2` prefix as well so there are no coordination issues around deployment? Particularly today when I'm supposed to be concentrating on OP stuff it would be good if I didn't block getting this merged and deployed.

Reply:
Good call!